### PR TITLE
Add description modification

### PR DESCRIPTION
**Pull request by YenguiSeddik**

### CODE REVIEW

File: src/components/dialogs/description-modification/description-modification-dialogue.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+interface IDescriptionModificationDialogue {
+    elementUuid: string;
+    description: string;
+    open: boolean;
+    onClose: () => void;
+}
+
+const schema = yup.object().shape({
+    [DESCRIPTION]: yup.string().nullable(),
```
Comment (AAJELLAL) [inline src/components/dialogs/description-modification/description-modification-dialogue.tsx:26]: The description field should not allow blank space and "" as values don't you think?

File: src/components/dialogs/description-modification/description-modification-dialogue.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    const { reset } = methods;
+
+    const onCancel = () => {
+        reset({
+            [DESCRIPTION]: '',
+        });
+        onClose();
+    };
+
+    const onSubmit = (data: { description: string }) => {
```
Comment (AAJELLAL) [inline src/components/dialogs/description-modification/description-modification-dialogue.tsx]: It would be better if you use useCallback.

**Review by AAJELLAL — CHANGES_REQUESTED** [review]
When I set whitespace as value a snackError is shown with the this message : 
403 Forbidden

File: src/components/dialogs/description-modification/description-modification-dialogue.tsx (hunk 1 above)
Comment (YenguiSeddik) [reply src/components/dialogs/description-modification/description-modification-dialogue.tsx:26]: Description is optionnal, so i think it should be allowed

**Review by souissimai — COMMENTED** [review]
Maybe you should truncating tooltips when the description text is too long. Please check this with PO, otherwise test ok and code ok

**Review by AAJELLAL — APPROVED** [review]

**Review by souissimai — APPROVED** [review]

**Review by souissimai — APPROVED** [review]

**Pull request merged**